python/_dynet.pyx

```diff
@@ -598,11 +604,12 @@ cdef class Expression: #{{{
     def __str__(self):
         return "exprssion %s/%s" % (<int>self.vindex, self.cg_version)
 
-    def __getitem__(self, int i):
-        return pick(self, i)
-
-    def __getslice__(self, int i, int j):
-        return pickrange(self, i, j)
+    # __getitem__ and __getslice__ in one for python 3 compatibility
```

Comment:
I trained a SimpleRNNNetwork from https://talbaumel.github.io/attention/ as a benchmark. I ran it in python2 multiple times with and without that change.

The difference in runtime seems negligible (up to 0.5%).